### ISSUE TITLE
Audio compressor and reset player not appearing

### ISSUE THREAD

**Issue by rmcin329** [FrankerFaceZ/FrankerFaceZ#1425]
**Describe the bug**

**To Reproduce**
1. Go to '...' Any stream on twitch

**Expected behavior**
I expect to see the audio compressor, and the reset player options.

**Screenshots**

**Environment (please complete the following information):**
 - OS: Windows 11 Pro
 - Browser Firefox
 - Version 119.0
 - Other Extensions: Ublock origin, bttv, cookie autodelete, Popup blocker for FF: Poper Blocker, norton safe web.

**FFZ Logs (via FFZ Control Center > Home > Feedback >> Log)**:

**Additional context**
Things were fine for most of the day then the mentioned items suddenly stopped appearing in streams. I have tried clearing cookies and cache, and restarting the pc.

**Comment by CodyCTops** [comment]
Just wanted to contribute that I also ran into this. A little testing and I noticed that:

1) opening the FFZ settings within twitch and toggling the "Enable Audio Compressor" setting off and then back on makes the compressor button re-appear. A page reload will cause the button to disappear again.

2) watching the HTML code I noticed that the <div class="ffz--player-comp tw-inline-flex tw-relative ffz-il-tooltip__container"></div> does not auto populate on initial page request or a page reload. It does however show up in the inspector after disabling the compressor and re-enabling it from the FFZ twitch page settings.
